fiboa_cli/convert_utils.py

```diff
@@ -246,11 +251,11 @@ def download_files(self, uris, cache_folder=None):
                 elif py7zr.is_7zfile(cache_file):
                     with py7zr.SevenZipFile(cache_file, 'r') as sz_file:
                         sz_file.extractall(zip_folder)
-                elif name.endswith(".rar"):
+                elif rarfile.is_rarfile(cache_file):
```

Comment:
I think there was an issue with this method for the SI converter, but not sure. As there's no test for SI, can you check whether the SI converter still works fine with this?

Reply:
Will add test for SI...